TombEngine/Game/Lara/lara_helpers.h

```diff
@@ -26,7 +26,10 @@ void DoLaraTightropeLean(ItemInfo* item);
 void DoLaraTightropeBalanceRegen(ItemInfo* item);
 void DoLaraFallDamage(ItemInfo* item);
 
+LaraInfo& GetLaraInfo(ItemInfo& item);
+const LaraInfo& GetLaraInfo(const ItemInfo& item);
```

Comment:
I'm not sure about this overloading, wouldn't the const in the returning type forbid the modification of LaraInfo? I understand the parametter const ItemInfo& is to explain the item won't be changed in this function, so it get more optimized by the compiler, but I don't know about the const on the returning.

I think that const should be in the destiny variable.

```c++
const auto myLaraInfo = GetLaraInfo ( LaraItem); //LaraItem enters as const in the GetLaraInfo function
```

Instead of
```c++
auto myLaraInfo = GetLaraInfo ( LaraItem); //LaraItem enters as const in the GetLaraInfo function
```

Reply:
Ah yes, I recently learned about this. :D You are right. The `const` version of the function doesn't need to exist. In general, the `const`ing here was meant as temporary scaffolding for potentially better object structuring in the future, but we don't need too much of it.

Reply:
Wait no, I still need it. It's just the `const` that's unnecessary.